docs/security.md

```diff
@@ -63,10 +65,42 @@ We use a number of static analysis tools to identify security risks in developme
 
 All development laptops have encrypted hard drives.
 
-## Bug reports
+## How to report a security vulnerability
+
+If you think that you have found a security issue, please email us at <a href="mailto:security@sourcegraph.com">security@sourcegraph.com</a>. We will reply to high quality reports within 1 business day to acknowledge that we received them and will strive to send you regular updates on our progress until the issue is resolved. You may request an update by replying to the existing email thread. We will read, but may not respond to or issue bounties for low quality or spammy reports (e.g. those produced by automated tooling).
```

Comment:
```suggestion
If you think that you have found a security issue, please email us at <a href="mailto:security@sourcegraph.com">security@sourcegraph.com</a>. We will reply to all reports within 1 business day to acknowledge that we received them and will strive to send you regular updates on our progress until the issue is resolved. You may request an update by replying to the existing email thread. We will read, but may not respond to or issue bounties for low quality or spammy reports (e.g. those produced by automated tooling).
```

I think this wording is better because it implies that our low-quality report handling is the exception.

Reply:
"All" seems like a direct contradiction, so I will just drop "high quality"

```suggestion
If you think that you have found a security issue, please email us at <a href="mailto:security@sourcegraph.com">security@sourcegraph.com</a>. We will reply to reports within 1 business day to acknowledge that we received them and will strive to send you regular updates on our progress until the issue is resolved. You may request an update by replying to the existing email thread. We will read, but may not respond to low quality or spammy reports (e.g. those produced by automated tooling).
```